connection.go

```diff
@@ -405,7 +406,12 @@ func (c *Connection) ReadOp() (_ context.Context, op interface{}, _ error) {
 
 		// Choose an ID for this operation for the purposes of logging, and log it.
 		if c.debugLogger != nil {
-			c.debugLog(inMsg.Header().Unique, 1, "<- %s", describeRequest(op))
+			debugMsg := describeRequest(op)
+			c.debugLog(inMsg.Header().Unique, 1, "<- %s", debugMsg)
+			if strings.Contains(debugMsg, "NO PID FOR OP") {
+				// Added for debugging temporarily.
+				panic("==NO PID FOR OP==")
+			}
```

Comment:
@dotslash Shall we include this in the final PR or this is for local debugging only?

Reply:
Remove.